starlette/passwords.py

```diff
@@ -0,0 +1,191 @@
+import abc
+import base64
+import binascii
+import hashlib
+import hmac
+import random
+from typing import Any, Callable, Optional, Sequence
+
+from starlette.concurrency import run_in_threadpool
+
+try:
+    import argon2
+except ImportError:  # pragma: nocover
+    argon2 = None
+
+try:
+    import bcrypt
+except ImportError:  # pragma: nocover
+    bcrypt = None
+
+ALNUM_CHARS = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+
+class PasswordHasher(abc.ABC):
+    algorithm: str = ""
+    rounds: int = 150000
```

Comment:
Is that a sensible value?

Reply:
do you mind to set default to null and override this on per-class basis?